tests/ops/models/test_detection_discovery.py

```diff
@@ -206,6 +207,40 @@ def test_create_monitor_config(
         assert mc_connection_config.key == connection_config.key
         assert mc_connection_config.connection_type == connection_config.connection_type
 
+        assert mc.databases == ["db1", "db2"]
+        assert mc.excluded_databases == []
+
+    def test_update_monitor_config_fails_with_conflicting_dbs(
+        self, db: Session, create_monitor_config, connection_config: ConnectionConfig
+    ) -> None:
+        """ """
+        with pytest.raises(Exception):
```

Comment:
nit: can we check specifically for ValueError here rather than exception?

Reply:
thank you, nice catch: https://github.com/ethyca/fides/pull/5080/commits/e5d32d7b249da0b39fe2d431188186132ae12e29